packages/components/inputs/rich-text-input/src/editor.styles.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Is this eslint disable comment necessary? It will be nice to remove it if it is not.